backend/test/controllers/bucket_test.ts

```diff
@@ -91,5 +93,31 @@ import {
     )
   }
 
+  @test async 'can be returned as page repositories'() {
+    const new_user = createNewUser(this.testUserData);
+    const new_bucket = createNewBucket(this.testBucketData(new_user._id));
+
+    await saveUser(new_user).then(saved_user =>
```

Comment:
Well, in scala anytime we do an Await we have a timeout set so that we never wait for a promose that never returns. Might be a "Nice to Have"